aws/templates/application/application_lambda.ftl

```diff
@@ -204,19 +204,16 @@
                 [#list solution.Alerts?values as alert ]
 
                     [#assign dimensions=[] ]
+                    [#assign metricName = alert.Metric.Name ]
 
                     [#switch alert.Metric.Type]
                         [#case "LogFilter" ]
-                            [#assign dimensions +=
-                                [
-                                    {
-                                        "Name" : "LogGroupName",
-                                        "Value" : logGroupName
-                                    }
-                                ]
-                            ]
-                        [#break]
-                    [/#switch]
+                            [#-- TODO: Ideally We should use dimensions for filtering but they aren't available on Log Metrics --]
+                            [#-- feature requst has been reaised... --]
+                            [#-- Instead we name the logMetric with the function name and will use that --]
+                            [#assign metricName = formatName(alert.Metric.Name, fnName) ]
```

Comment:
metricName = formatName(metricName, fnName)

Also shouldn't fnNAme came first? Not sure how this affects sorting in the console...

Reply:
IT would let you see the same metrics grouped together for each environment. This would be the way you look for them with dimensions, you go to the metric then filter it based on dimension to get the details you are after